README.md

```diff
@@ -85,7 +85,8 @@ See `dotnet-gitchanges\appsettings.json` for the default settings file. Any sett
       "OverrideSource": "someOverrideSource.txt"
     },
     "FileSource": "someHistorialChanges.txt",
-    "MultiProject": false
+    "MultiProject": false,
+    "UseGitTags": false

```

Comment:
Instead of "UseGitTags", lets call it "VersionFromTag"

Reply:
I think it should probably be "VersionFromGitTag" - since otherwise someone might confuse it with the tags that can be set on commits for which section in the produced changelog?